Formula/dashlane-cli.rb

```diff
@@ -0,0 +1,54 @@
+require "language/node"
+
+class DashlaneCli < Formula
+  desc "Command-line interface for Dashlane"
+  homepage "https://dashlane.com"
+  url "https://github.com/Dashlane/dashlane-cli/archive/refs/tags/v1.4.0.tar.gz"
+  sha256 "9c3dd05f22cda71f33c3148766e9573320ff150b2203d659dd4a7442f312802a"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "node@16" => :build
+  depends_on "yarn" => :build
+
+  on_macos do
+    # macos requires binaries to do codesign
+    depends_on xcode: :build
+    # macos 12+ only
+    depends_on macos: :monterey
+  end
+
+  on_linux do
+    depends_on "glibc" if Formula["glibc"].any_version_installed?
```

Comment:
just a clueless attempt to make the linux runner work

Reply:
didn't change anything, also see my investigation below, the problem is rather on the interpreter